web/src/popovers.js

```diff
@@ -1083,8 +1083,15 @@ export function any_active() {
 export function hide_all_except_sidebars(opts) {
     $(".has_popover").removeClass("has_popover has_actions_popover has_emoji_popover");
     if (!opts || !opts.not_hide_tippy_instances) {
-        // hideAll hides all tippy instances (tooltips and popovers).
-        hideAll();
+        if (opts && opts.not_hide_tippy_dropdowns) {
+            // don't hide gear menu and personal menu dropdows
+            hideAll({
+                exclude: popover_menus.get_personal_menu_instance(),
+            });
```

Comment:
We can just check `opts.not_hide_tippy_dropdowns` here since `opts` is defined. I think this if fine for now but let's add a comment that when we add "gear menu" then we should add a function which gets the current open tippy dropdown for us which is just either personal menu or gear menu.